Examples/Scripts/PreFreeSurferPipelineBatchNHP.sh

```diff
@@ -0,0 +1,337 @@
+            AvgrdcSTRING="TOPUP" #Averaging and readout distortion correction methods: "NONE" = average any repeats with no readout correction "FIELDMAP" = average any repeats and use field map for readout correction "TOPUP" = Use Spin Echo FieldMap
+        elif [[ $(imtest $StrucMagnitudeInputName) = 1 && $(imtest $StrucPhaseInputName) = 1 && -n $StrucDwelltime ]] ; then
+            MagnitudeInputName="${StudyFolder}/${Subject}/RawData/${StrucMagnitudeInputName}" #Expects 4D magitude volume with two 3D timepoints or "NONE" if not used
+            PhaseInputName="${StudyFolder}/${Subject}/RawData/${StrucPhaseInputName}" #Expects 3D phase difference volume or "NONE" if not used
+            DiffTE="${DiffTE:-2.46}"   # default for 3T
+            AvgrdcSTRING="FIELDMAP" #Averaging and readout distortion correction methods: "NONE" = average any repeats with no readout correction "FIELDMAP" = average any repeats and use field map for readout correction "TOPUP" = Use Spin Echo FieldMap
+        else
+            SpinEchoPhaseEncodeNegative="NONE"
+            SpinEchoPhaseEncodePositive="NONE" 
+            AvgrdcSTRING="NONE"
+        fi
+    else
+        SpinEchoPhaseEncodeNegative="NONE" #For the spin echo field map volume with a negative phase encoding direction (LR in HCP data), set to NONE if using regular FIELDMAP
+        SpinEchoPhaseEncodePositive="NONE" #For the spin echo field map volume with a positive phase encoding direction (RL in HCP data), set to NONE if using regular FIELDMAP
+        TE="NONE" # "2.46" delta TE in ms for field map or "NONE" if not used
+        SEEchoSpacing="NONE" # Echo Spacing or SEEchoSpacing of SE Field Map image (or "NONE" if not used) = 1/(BandwidthPerPixelPhaseEncode * # of phase encoding samples): DICOM field (0019,1028) = BandwidthPerPixelPhaseEncode, DICOM field (0051,100b) AcquisitionMatrixText first value (# of phase encoding samples)
+        SEUnwarpDir="NONE" # x or y (minus or not does not matter) "NONE" if not used
+        UnwarpDir="NONE"
+        AvgrdcSTRING="NONE" #Averaging and readout distortion correction methods: "NONE" = average any repeats with no readout correction "FIELDMAP" = average any repeats and use field map for readout correction   "TOPUP" = Use Spin Echo FieldMap"${
+        TopupConfig="NONE" #Config for topup or "NONE" if not used
+        T1wSampleSpacing="${T1wSampleSpacing:-NONE}" #"0.0000150" DICOM field (0019,1018) in s or "NONE" if not used
+        T2wSampleSpacing="${T2wSampleSpacing:-NONE}" #"0.0000036" DICOM field (0019,1018) in s or "NONE" if not used
+        AvgrdcSTRING="NONE"
+    fi
+
+    if [[ $Gradient = "" || $Gradient = "NONE" || $Gradient = "None" ]] ; then
+        GradientDistortionCoeffs="NONE"               #Location of Coeffs file or "NONE" to skip
+    else
+        GradientDistortionCoeffs=${HCPPIPEDIR_Config}/coeff_${Gradient}.grad
```

Comment:
Let's just make this a full absolute path to the gradient coefficents without an extra variable.